content/accessible/labels-and-text-alternatives/index.md

```diff
@@ -52,13 +52,14 @@ DevTools Accessibility Reference
 </a>.
 </div>
 
-Whether you're looking at an `img` with `alt` text, or an `input` with a
-`label`, you'll soon discover that all of these things result in the same
-outcome: giving an element its accessible name.
+Whether you're looking at an `img` with `alt` text or an `input` with a
+`label`, all of these attributes result in the same outcome: giving an 
```

Comment:
```suggestion
`label`, all of these scenarios result in the same outcome: giving an 
```

suggest maybe using "scenario" because `label` is a tag not an attribute